sound/soc/sof/intel/hda-codec.c

```diff
@@ -173,24 +174,23 @@ static int hda_codec_probe(struct snd_sof_dev *sdev, int address,
 		 * other return codes without modification
 		 */
 		if (ret == 0)
-			goto error;
+			ret = -ENOENT;
 	}
 
-	return ret;
-
 error:
```

Comment:
Oops, the "error:" label is no longer needed. Plus a typo in commit, will reupload in a minute.

Reply:
why? line 160 still calls goto error no?

Reply:
ack, but it's confusing to have it labeled error. I'll rename to "out".